### PR TITLE
GraphQL: fix LegalEntities division addresses, added owner

### PR DESCRIPTION
**Pull request by PavelVesnin**

### CODE REVIEW

File: apps/graphql/lib/graphql_web/loaders/division_loader.ex [hunk 1]
```diff
@@ -28,7 +28,7 @@ defmodule GraphQLWeb.Loaders.DivisionLoader do
     |> offset(^offset)
   end
 
-  def query(_queryable, _args), do: from(l in Division)
+  def query(entity, _args), do: from(l in entity)
```
Comment (kiurchv) [inline apps/graphql/lib/graphql_web/loaders/division_loader.ex]:
```suggestion
  def query(queryable, _args), do: queryable
```

File: apps/graphql/lib/graphql_web/loaders/employee_loader.ex [hunk 2]
```diff
@@ -30,6 +30,20 @@ defmodule GraphQLWeb.Loaders.EmployeeLoader do
...
 
+  def load_owner(parent, args, %{context: %{loader: loader}}) do
+    args =
+      Map.merge(args, %{
+        employee_type: Employee.type(:owner),
+        is_active: true
+      })
+
+    loader
+    |> Dataloader.load(__MODULE__, {:employee, args}, parent)
```
Comment (kiurchv) [inline apps/graphql/lib/graphql_web/loaders/employee_loader.ex]:
```suggestion
    |> Dataloader.load(PRM, {:employee, args}, parent)
```

File: apps/graphql/lib/graphql_web/loaders/employee_loader.ex [hunk 3]
```diff
@@ -30,6 +30,20 @@ defmodule GraphQLWeb.Loaders.EmployeeLoader do
...
+    |> ResolutionHelper.on_load(fn loader ->
+      {:ok, Dataloader.get(loader, __MODULE__, {:employee, args}, parent)}
```
Comment (kiurchv) [inline apps/graphql/lib/graphql_web/loaders/employee_loader.ex]:
```suggestion
      {:ok, Dataloader.get(loader, PRM, {:employee, args}, parent)}
```

Comment (kiurchv) [reply apps/graphql/lib/graphql_web/loaders/employee_loader.ex]: Also, add on top of module: ```alias GraphQLWeb.Loaders.PRM```

File: apps/graphql/lib/graphql_web/schema/division_types.ex [hunk 4]
```diff
@@ -27,7 +31,7 @@ defmodule GraphQLWeb.Schema.DivisionTypes do
     # embed
 
     field(:phones, non_null(list_of(:phone)))
-    field(:addresses, non_null(list_of(:address)))
+    field(:addresses, non_null(list_of(:address)), resolve: dataloader(DivisionLoader))
```
Comment (kiurchv) [inline apps/graphql/lib/graphql_web/schema/division_types.ex]: use GraphQLWeb.Loaders.PRM here

File: apps/graphql/lib/graphql_web/loaders/employee_loader.ex [hunk 5]
```diff
@@ -30,6 +30,20 @@ defmodule GraphQLWeb.Loaders.EmployeeLoader do
 
   def query(_queryable, _args), do: from(l in Employee)
 
+  def load_owner(parent, args, %{context: %{loader: loader}}) do
```
Comment (kiurchv) [inline apps/graphql/lib/graphql_web/loaders/employee_loader.ex]: move this function to `GraphQLWeb.Resolvers.LegalEntity`

File: apps/graphql/lib/graphql_web/schema/legal_entity_types.ex [hunk 6]
```diff
@@ -115,7 +115,8 @@ defmodule GraphQLWeb.Schema.LegalEntityTypes do
     field(:medical_service_provider, non_null(:msp))
 
     # relations
-    #    field(:owner, :employee, resolve: dataloader(EmployeeLoader))
+    field(:owner, :employee, resolve: &EmployeeLoader.load_owner/3)
```
Comment (kiurchv) [inline apps/graphql/lib/graphql_web/schema/legal_entity_types.ex]:
```suggestion
    field(:owner, :employee, resolve: &LegalEntityResolver.load_owner/3)
```

File: apps/graphql/lib/graphql_web/loaders/employee_loader.ex (hunk 3 above)
Comment (PavelVesnin) [reply apps/graphql/lib/graphql_web/loaders/employee_loader.ex]: but your PR is not merged, there is no `GraphQLWeb.Loaders.PRM` loader

**Pull request merged**